composer.json

```diff
@@ -15,13 +15,18 @@
 		"php": ">=5.6"
 	},
 	"require-dev": {
-		"dealerdirect/phpcodesniffer-composer-installer": "^0.6",
 		"roave/security-advisories": "dev-master",
-		"wptrt/wpthemereview": "^0.2"
+		"wptrt/wpthemereview": "*",
+		"dealerdirect/phpcodesniffer-composer-installer": "0.6",
+		"php-parallel-lint/php-parallel-lint": "^1.2"
 	},
 	"scripts": {
-		"check-cs": ["@php ./vendor/squizlabs/php_codesniffer/bin/phpcs -w"],
-		"fix-cs": ["@php ./vendor/squizlabs/php_codesniffer/bin/phpcbf"]
+		"check:cs": ["@php ./vendor/squizlabs/php_codesniffer/bin/phpcs -p"],
+		"check:compatibility": ["@php ./vendor/squizlabs/php_codesniffer/bin/phpcs -p --standard=phpcs.xml,PHPCompatibilityWP"],
+		"check:theme-review": ["@php ./vendor/squizlabs/php_codesniffer/bin/phpcs -p --standard=phpcs.xml,WPThemeReview"],
```

Comment:
These don't make any sense as they will just result in the same as `check:cs` as both rulesets are already included in the `phpcs.xml` file.

Reply:
Thank you!